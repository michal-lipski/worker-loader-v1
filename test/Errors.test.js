import { pitch } from '../src';

describe('Errors', () => {
  test('Loader Error', () => {
    const error = () => pitch.call({ webpack: false, query: {} });

    expect(error).toThrow();
    expect(error).toThrowErrorMatchingSnapshot();
  });

  test('Validation Error', () => {
    const error = () => pitch.call({ query: { name: 1 } });

    expect(error).toThrow();
    expect(error).toThrowErrorMatchingSnapshot();
  });
});
