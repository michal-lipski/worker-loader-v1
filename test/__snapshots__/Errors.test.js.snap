// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors Loader Error 1`] = `
"Worker Loader

This loader is only usable with webpack
"
`;

exports[`Errors Validation Error 1`] = `
"Validation Error

Worker Loader Invalid Options

options.name should be string
"
`;
